thingif/src/main/java/com/kii/thingif/command/Action.java

```diff
@@ -30,5 +24,12 @@
  *
  */
 public interface Action {
+    /**
```

Comment:
The problem is who should validate value of member fields.

I think a class having member fields should validate the member fields because if a class does not validate the member fields, we must validate the member fields anywhere we use it. It is redundant.